UI/RemoteISOScreen.cpp

```diff
@@ -189,6 +189,17 @@ static bool FindServer(std::string &resultHost, int &resultPort) {
 	Buffer result;
 	int code = 500;
 
+	// Not sure why this is needed here, but windows fails without it.
```

Comment:
Well, you need to call net::Init() somewhere on Windows. Kind of odd to call it in here though... 

I'm going to clean our net init up before we merge this.